source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -190,20 +190,21 @@ def tag_ecologic(resp):
             if j.co2_emission.value < resp.car_co2_emission.value * 0.5:
                 j.tags.append('ecologic')
 
-def tag_direct_path(resp):
+def _tag_direct_path(responses):
     street_network_mode_tag_map = {response_pb2.Walking: ['non_pt_walking'],
                                    response_pb2.Bike: ['non_pt_bike']}
-    for j in resp.journeys:
-        if all(s.type != response_pb2.PUBLIC_TRANSPORT for s in j.sections):
-            j.tags.extend(['non_pt'])
+    for r in responses:
+        for j in r.journeys:
```

Comment:
It doesn't work. The one which works is not so good-looking for me

for j in itertools.chain.from_iterable(r.journeys for r in responses)